pinchy/src/events.rs

```diff
@@ -413,15 +413,15 @@ pub async fn handle_event(event: &SyscallEvent, formatter: Formatter<'_>) -> any
             argf!(sf, "fd: {}", data.fd);
             argf!(sf, "offset: 0x{:x}", data.offset);
 
-            finish!(sf, ret_str);
+            finish_raw!(sf, ret_str);
```

Comment:
looking at the 2 cases where you use finish_raw!, I am confident they can just be added to the return formatting structure along with the rest instead of being handled in a special way